Source/Scene/Cesium3DTileset.js

```diff
@@ -109,6 +111,8 @@ define([
      * @param {Boolean} [options.show=true] Determines if the tileset will be shown.
      * @param {Matrix4} [options.modelMatrix=Matrix4.IDENTITY] A 4x4 transformation matrix that transforms the tileset's root tile.
      * @param {Number} [options.maximumScreenSpaceError=16] The maximum screen-space error used to drive level-of-detail refinement.
+     * @param {Number} [options.maximumNumberOfLoadedTiles=256] The maximum number of tiles to load.
```

Comment:
Do we still need `maximumNumberOfLoadedTiles`?  When would it be used now that we have `maximumMemoryUsage`?